testing/deploy_kubeflow.py

```diff
@@ -47,7 +47,7 @@ def deploy_kubeflow(test_case):
   """Deploy Kubeflow."""
   args = parse_args()
   test_dir = test_case.test_suite.test_dir
-  namespace = args.namespace
+  namespace = "default"
```

Comment:
why do we use default?

Reply:
Namespace isn't being passed anywhere, so all the components are deployed in the default namespace. See the logs: http://testing-argo.kubeflow.org/workflows/kubeflow-test-infra/kubeflow-presubmit-kubeflow-e2e-minikube-1851-379d825-4045-4620?tab=workflow&nodeId=kubeflow-presubmit-kubeflow-e2e-minikube-1851-379d825-4045-4620-2307009077&sidePanel=logs%3Akubeflow-presubmit-kubeflow-e2e-minikube-1851-379d825-4045-4620-2307009077%3Amain

Reply:
Isn't namespace being used at line 52 to setup the app?

Reply:
Right, but all the components are deployed under the default namespace. On line 106 we look for the deployment, so if we pass the wrong namespace then it will fail. We can either use the default namespace, or figure out how to actually pass the namespace to each component.

Reply:
Maybe the right fix is to change line 83 to "namespace"?

Reply:
I think its only minikube that is using deploy_kubeflow.py.

I would expect deploy_utils.setup_kubeflow_ks_app to set the namespace of the environment which should correctly set the namespace for each component.

So we shouldn't need to set namespace when we generate each component.

So removing namespace as an argument seems correct.

Reply:
Can we pass in a value of "default" for --namespace rather than ignoring the argument and hardcoding "default"